### PR TITLE
GetProjects stats

### PR DESCRIPTION
**Pull request by PauuloG**

### CODE REVIEW

File: src/models/Stats/index.js [hunk 1]
```diff
@@ -1,28 +1,33 @@
...
-    projectsCount: ?number
-  ) {
-    this.missingTranslationsCount = missingTranslationsCount
-    this.newKeysCount = newKeysCount
-    this.completePercentage = completePercentage
+  constructor(args: StatsConstructor) {
+    this.missingTranslationsCount = args.missingTranslationsCount
+    this.newKeysCount = args.newKeysCount
+    this.completePercentage = args.completePercentage
+    this.translationKeysCount = args.translationKeysCount
```
Comment (Tom-Bonnike) [inline src/models/Stats/index.js]: FYI that’s probably better:
```js
constructor({ missingTranslationsCount, newKeysCount … }: StatsConstructorArgs) {
  this.missingTranslationsCount = missingtranslationsCount
  this.newKeysCount = newKeysCount
  …
)
```

File: src/models/Stats/index.js [hunk 2]
```diff
@@ -1,28 +1,33 @@
...
 
-    if (localesCount) {
-      this.localesCount = localesCount
+    if (args.localesCount) {
+      this.localesCount = args.localesCount
```
Comment (Tom-Bonnike) [inline src/models/Stats/index.js]: again, it’s fine setting them to undefined, the `if(…)` guards are not really necessary.

File: src/models/Project/index.js [hunk 3]
```diff
@@ -63,6 +66,70 @@ class Project extends Model {
...
+      .pluck('count')
+      .first()
+
+    return new Stats({
+      missingTranslationsCount,
+      newKeysCount,
+      completePercentage,
+      translationKeysCount,
+      localesCount,
+      projectsCount: null
```
Comment (Tom-Bonnike) [inline src/models/Project/index.js]: why always setting it to null?

Comment (PauuloG) [reply src/models/Project/index.js]: Because those are project stats (so not interested in getting that particular property at this point)

**Pull request merged**